### DISCUSSION TITLE
Virtual Tutor

### DISCUSSION THREAD

**Discussion by HarshYadavade** [monkeytypegame/monkeytype#4562] [Ideas] [answered]
Introduce an interactive virtual tutor that guides users through proper typing techniques and offers tips for improvement.

**Comment by Miodec** [comment, marked as answer]
Thats currently out of scope. Im focusing on providing the best tool, and for now its up to the user to use it to its best potential. Typing lessons might be added in the future.

**Comment by unrealapex** [comment]
As Miodec mentioned, this is out of the scope of the project. [typing.com](https://www.typing.com/) is a good website to learn. You can also find resources to improve at typing on YouTube and Reddit. Many people on the Monkeytype Discord server are also happy to provide advice if you need any. Ask away in the `#typing` channel.
